.gitignore

```diff
@@ -6,3 +6,4 @@ dist/
 .coverage
 .coverage.*
 .tox
+.vscode
```

Comment:
A quick note: I won't require this to be removed but I would prefer not to add it. I may remove it in the future (but I'm not such a curmudgeon that I'll do it right after merging "just because" :wink: ).

My opinion is that `.gitignore` files per repo should cover files produced by that repo, and that a global gitignore should be used for developer-specific things (`.idea`, `.vscode`, `.DS_Store`, etc).